### ISSUE TITLE
Sentry Turret noises seem to be centered on the player

### ISSUE THREAD

**Issue by MuteTiefling** [TeamPneumatic/pnc-repressurized#519]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

### Minecraft Version
1.15.2
### Forge Version
31.2
### Mod Version
1.2.2-30
### Describe your problem, including steps to reproduce it

Or they're just really loud. I haven't messed with their volume at all yet.

Still, I have a sentry turret set up in part of my base where I haven't quite 'fixed' the bat problem. The turret takes them out when they spawn. But I've noticed that even when I'm a few chunks away from that area the turret sounds like it's right on top of me when it spins up.

**Comment by desht** [comment]
I had noticed a bit of excess volume there, yeah. Was on my list of things to check into...

**Comment by desht** [comment]
Just did some investigation, and the sound is definitely being played at the TE's location, not the player's.  But you're right that it does seem much too loud, and it doesn't appear to attenuate at all with distance.  The default volume is also pretty low for the minigun, at 0.3, which doesn't give much room for lowering it.

I need to fire up Audacity and take a closer look at the minigun audio clip; it is definitely a very loud sound, but I'm not sure why Minecraft isn't doing any kind of distance-based attenuation on it at all.

**Comment by desht** [comment]
So here's a fun fact I learned today: if sounds are in stereo, Minecraft does not do any attenuation of them at all.  Not even a bit.  But if converted to mono, everything's properly attenuated by distance.  And it turns out a lot (most) of PNC's sounds are currently in stereo.

Pretty sure I can convert them all to mono with no ill effects (and also change default volume levels in the config, since a lot of levels were pushed low precisely because sounds were played too loudly).  The one thing you (and other players) might need to do is raise those levels in your config if you're upgrading since a lot of stuff may sound too quiet now.

**Comment by desht** [comment]
When you get a chance, can you try out build 34 please?

If you're upgrading from an existing install, you may want to play with the volumes in `config/pneumaticcraft-client.toml` since sounds may feel a lot quieter now.  Part of the reason default values were turned down was because sounds were too loud, which should no longer be an issue.  I'm using these settings right now:

```
[sound]
        #Volume level of the Jet Boots in normal flight mode
        #Range: 0.0 ~ 2.0
        jetboots_volume = 1.0
        #Volume level of the hand-held Minigun
        #Range: 0.0 ~ 2.0
        minigun_volume_held = 0.5
        #Volume level of the Sentry Turret's Minigun
        #Range: 0.0 ~ 2.0
        minigun_volume_sentry_turret = 1.0
        #Volume level of the Elevator *clunk* while starting/stopping
        #Range: 0.0 ~ 2.0
        elevator_volume_start_stop = 1.0
        #Volume level of drone-mounted Miniguns
        #Range: 0.0 ~ 2.0
        minigun_volume_drone = 1.0
        #Volume level of the Jet Boots when in Builder Mode
        #Range: 0.0 ~ 2.0
        jetboots_volume_builder_mode = 0.3
        #Volume level of air leaks from unconnected tubes/machines. Beware: turning this off could lead to undetected leaks wasting pressure forever!
        #Range: 0.0 ~ 2.0
        air_leak_volume = 1.0
        #Volume level of the Elevator while running
        #Range: 0.0 ~ 2.0
        elevator_volume_running = 1.0
```

As you can see, most sounds should now be fine at a volume of 1.0 (down to personal preference of course).  I turned down the hand-held minigun to 0.5, since it's always played right at the player's location, and the jet boots builder mode to 0.3.

**Comment by MuteTiefling** [comment]
Heh, minecraft is so goofy sometimes. Yes, this is about a million times better. I'm about 3 chunks away and can still see it shooting, but the sound dropped off sharply as I moved away.

**Comment by desht** [comment]
Fixed in 1.3.1 release
